larray/core/session.py

```diff
@@ -318,12 +462,61 @@ def copy(self):
         return Session(self._objects)
 
     def keys(self):
+        """
+        Returns the list of names of the array objects in the session
+
+        Returns
+        -------
+        list of str
+
+        Examples
+        --------
+        >>> s = Session(OrderedDict([('arr1', ndtest((3, 3))), ('arr2', ndtest((6, 4))), ('arr3', ndtest((2, 8)))]))                      
```

Comment:
Initialize session with arr2, arr1, arr3 (ie names not alphabetically ordered) so that we see keys keep order?